### DISCUSSION TITLE
Add Chrome Version

### DISCUSSION THREAD

**Discussion by bolach** [cpriest/SnapLinksPlus#508] [Ideas]
Hello,
I'm a chromium user because of some exclusive extensions. I would prefer firefox otherwise.
I want to suggest a version of Snap Links to Chrome. I saw on snap links official page a link to chrome extensions page, but it is not working, so i guess it is not updated anymore.

I don't know why, maybe chrome limitation, but none of the extensions similar to snap links on chrome works perfectly like this one does.
When i want to use this, i open Firefox and always does the job!

Thank you for this great job!

**Comment by cpriest** [comment]
I did have a chrome version when I had briefly switched to Chrome, but I've switched back because I don't like Google's dominance w/ Chrome at this point.

Besides that, I really don't have the time to put much new into this project so I'm afraid an official chrome version won't be forthcoming without someone holding the torch.
